src/twisted/trial/_dist/workerreporter.py

```diff
@@ -45,8 +45,8 @@ async def addError(
     :param frames: The lines of the traceback associated with the error.
     """
 
-    errorStreamId = await stream(amp, chunk(error, 2 ** 16 - 1))
-    framesStreamId = await stream(amp, iter(frames))
+    errorStreamId = await stream(amp, chunk(error.encode("utf-8"), 2 ** 16 - 1))
```

Comment:
`2 ** 16 - 1` is fine for now.
I guess that in an ideal world, this would be a constant that can be shared between the producer and consumer.

Reply:
I'm not sure if I follow.  The consumer can always figure this out by looking at the length of the chunks it receives. :)

Reply:
True. Thanks for the followup.

I was thinking to have the constant between this code and the AMP protocol.
As this whole chunking work is due to an AMP protocol limitation.

Reply:
Ahh yes I see.  I think AMP does have a constant for this,